organizationdomain/client.go

```diff
@@ -0,0 +1,122 @@
+// Package organizationdomain provides the Organization Domains API.
+package organizationdomain
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+)
+
+//go:generate go run ../cmd/gen/main.go
+
+const path = "/organizations"
+
+// Client is used to invoke the Organization Domains API.
+type Client struct {
+	Backend clerk.Backend
+}
+
+func NewClient(config *clerk.ClientConfig) *Client {
+	return &Client{
+		Backend: clerk.NewBackend(&config.BackendConfig),
+	}
+}
+
+type CreateParams struct {
+	clerk.APIParams
+	Name           string `json:"name" form:"name"`
+	OrganizationID string `json:"-" form:"-"`
```

Comment:
:wrench: This field should be removed. The organization ID is not part of the parameters to create an organization.